sounds/forms.py

```diff
@@ -118,15 +118,7 @@ def save(self):
             try:
                 source = Sound.objects.get(id=sid)
                 self.sound.sources.remove(source)
-
                 source.invalidate_template_caches()
-
-                # modify remix_group
-                send_mail_template(
-                    u'Sound removed as remix source', 'sounds/email_remix_update.txt',
-                    {'source': source, 'action': 'removed', 'remix': self.sound},
-                    user_to=source.user
```

Comment:
Because it's not very useful to notify about a sound being removed from a remix group. Just notifying when a sound is added is fine. I think I discussed this with Philip. No one is doing much remixing anyways...